cherrypy/lib/httputil.py

```diff
@@ -518,7 +518,8 @@ def encode_header_items(cls, header_items):
         transmitting on the wire for HTTP.
         """
         for k, v in header_items:
-            if not isinstance(v, six.string_types):
+            if not isinstance(v, six.string_types) and \
```

Comment:
Please use brackets instead. Escaping EOL looks ugly and we don't use it.

Reply:
And this probably only needs `binary_type` check.

Reply:
So basically, I applied the same fix (and same test) as in the original fix.